src/App.js

```diff
@@ -54,6 +57,181 @@ const App = () => {
           </div>
         </div>
       </section>
+      <section className="digits">
+        <h3 className="digits__title">Цифры и факты</h3>
```

Comment:
`h3` can not be bigger than `h2`
Switch them

Reply:
done